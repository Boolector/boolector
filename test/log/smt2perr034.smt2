(declare-fun f () (epected_underscore_or_Array
