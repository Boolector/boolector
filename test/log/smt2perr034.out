log/smt2perr034.smt2:1:20: expected '_' or 'Array' at 'epected_underscore_or_Array'
